user-interface/src/case-detail/panels/CaseDetailAuditHistory.tsx

```diff
@@ -12,6 +17,82 @@ export interface CaseDetailAuditHistoryProps {
 export default function CaseDetailAuditHistory(props: CaseDetailAuditHistoryProps) {
   const { caseHistory, isAuditHistoryLoading } = props;
 
+  function showCaseAssignmentHistory(history: CaseAssignmentHistory, idx: number) {
+    return (
+      <tr key={idx}>
+        <td>Staff</td>
+        <td data-testid={`previous-assignment-${idx}`}>
+          {history.before.length === 0 && <>(none)</>}
+          {history.before
+            .map((assignment) => {
+              return assignment.name;
+            })
+            .join(', ')}
```

Comment:
Should we use that function we created recently that prepares assignee names for display?

Reply:
If so, the after column should get the same treatment.

Reply:
Probably not here because we are not rendering a sentence.